case_studies/starnet/m2_results.ipynb

```diff
@@ -0,0 +1,1005 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
```

Comment:
Do we want to strip the output of the case studies notebooks? I'm not sure. All the output makes for large diffs and can make the repository large. On the other hand, it's nice for people to see the results without having to run it themselves.